apidocs/getNeighbors.md

```diff
@@ -0,0 +1,136 @@
+
+---
+### [getNeighbors](https://github.com/iotaledger/iri/blob/dev/src/main/java/com/iota/iri/service/API.java#L705)
+ [AbstractResponse](/javadoc/com/iota/iri/service/dto/abstractresponse/) getNeighborsStatement()
+
+Returns the set of neighbors you are connected with, as well as their activity count. 
```

Comment:
activity count -> activity statistics (or counters)
The activity counter is reset -> The acivity counters are reset


(from the perspective of the return values, not 'internal counters')